Cubical/Data/Int/Properties.agda

```diff
@@ -451,6 +827,311 @@ inj-+z {z} {l} {n} p = inj-z+ {z = z} (+Comm z l ∙ p ∙ +Comm n z)
 n+z≡z→n≡0 : ∀ n z → n + z ≡ z → n ≡ 0
```

Comment:
Not your fault, but it's bad to have lemma names named after arbitrary choices of variable names. Can you fix?

Reply:
I'm going to redo most of these in the later PR where I will revise all the Nat/Int properties to defer to their algebraic structure.